Client/Info.plist

```diff
@@ -31,8 +31,6 @@
 	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
-	<key>CFBundleName</key>
```

Comment:
Is this okay to remove? Or is this getting set anywhere else?

Reply:
Unused that i could find, and reportedly unused on iOS in general https://stackoverflow.com/questions/12146750/where-cfbundlename-is-being-used